### PR TITLE
Fix CSP style-src violation in Util.js

### PR DESCRIPTION
**Pull request by kamaldeepcheema-okta**
## Description:

- Fixing the CSP style-src violation in Util.js as per the guidelines here: https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2703983302/Identifying+Fixing+style-src+based+CSP+vulns#2.-style-in-attribute

Bacon: https://bacon-go.aue1e.saasure.net/commits?artifact=okta-signin-widget&branch=kc-fix-CSP-error&page=1&pageSize=6&sha=5c6c72a102b1b1010d403a9cb33e13e421b45cea&tab=main

## PR Checklist

- [ ] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-548150](https://oktainc.atlassian.net/browse/OKTA-548150)

### CODE REVIEW

File: test/testcafe/spec/IdentifyWithThirdPartyIdps_spec.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ test.requestHooks(logger, mockOnlyOneIdp)('should auto redirect to 3rd party IdP
 test.requestHooks(logger, mockOnlyOneIdp)('Direct auth: does not auto redirect to 3rd party IDP on initial load', async t => {
   const identityPage = await setupDirectAuth(t);
 
-  // await t.wait(100);
+  await t.wait(100);
```
Comment (kamaldeepcheema-okta) [inline test/testcafe/spec/IdentifyWithThirdPartyIdps_spec.js:146]: I noticed a lot of Bacon flakiness with this test, and uncommenting this seems to make it stable.

**Review by aarongranick-okta — CHANGES_REQUESTED** [review]
Can you add a test to assert the fix and also protect against regressions?

CSP headers are added to the test app here:
https://github.com/okta/okta-signin-widget/blob/42dc0dc9f843284b7ab1c7cfe73f098e0b5956da/test/app/webpack.config.js#L52
(for E2E tests)

and to the widget playground here:
https://github.com/okta/okta-signin-widget/blob/master/webpack.playground.config.js#L37
(For testcafe)

The current approach is to enforce CSP across all tests. Then a "negative" test is added to be sure that we are not getting false positives: https://github.com/okta/okta-signin-widget/blob/42dc0dc9f843284b7ab1c7cfe73f098e0b5956da/test/e2e/features/basic.feature#L22 This is currently only testing with eval:

https://github.com/okta/okta-signin-widget/blob/2d0e19a72c39d666d7c5b76ed984f0457dd7f5ff/test/app/src/TestApp.ts#L292 

We should test that both script-src and style-src are being enforced as expected.

File: test/testcafe/spec/IdentifyWithThirdPartyIdps_spec.js (hunk 1 above)
Comment (kamaldeepcheema-okta) [reply test/testcafe/spec/IdentifyWithThirdPartyIdps_spec.js:146]: Yeah I did try to debug this quite a bit locally and the first thing I tried was to wait for the form render with the social auth buttons but that didn't seem to help. Tbh, I wasn't quite sure why that didn't work and a specific wait for 100ms did. But I can try to investigate some more.

**Comment by sherizada-okta** [comment]
merged into this PR https://github.com/okta/okta-signin-widget/pull/2925

**Pull request closed without merging**